src/debug.cpp

```diff
@@ -34,7 +34,7 @@
 #endif
 
 #ifdef TILES
-#include <SDL.h>
+#include "sdl_wrappers.h"
```

Comment:
Why that? "sdl_wrappers.h" includes more than just "SDL.h" and this file does not use any of the additional stuff declared in "sdl_wrappers.h" only. So it basically just includes more files without any advantage.